apps/dataset-browser/src/app/[locale]/datasets/[id]/page.tsx

```diff
@@ -59,18 +59,14 @@ export default async function Details({params}: Props) {
     {
       name: t('metadata.license'),
       value: (
-        <a href={dataset.license.id} target="_blank" rel="noreferrer">
-          {dataset.license.name}
+        <a href={dataset.license?.id} target="_blank" rel="noreferrer">
+          {dataset.license?.name}
         </a>
       ),
     },
     {
       name: t('metadata.publisher'),
-      value: (
-        <a href={dataset.publisher.id} target="_blank" rel="noreferrer">
-          {dataset.publisher.name}
-        </a>
-      ),
+      value: dataset.publisher?.name,
```

Comment:
Just a note: I think users would like a link to the publisher, but we can move this to a feature request when users ask. If the data is absent, I agree with removing the link tag.

Reply:
The previous code assumed `dataset.publisher.id` pointed to a website, but it's actually an (linked data) identifier that doesn't have to resolve to a webpage that's useful to users. I agree - if users want to be able to go to a website, we should created a feature request.